skimage/measure/fit.py

```diff
@@ -282,6 +282,15 @@ def estimate(self, data):
         # to prevent integer overflow, cast data to float, if it isn't already
         float_type = np.promote_types(data.dtype, np.float32)
         data = data.astype(float_type, copy=False)
+        # normalize value range to avoid misfitting due to numeric errors if
+        # the relative distanceses are small compared to absolute distances
+        origin = data.mean(axis=0)
+        data = data - origin
+        scale = data.std()
```

Comment:
If we take the mean over `axis=0`. Shouldn't we do the same for `std()`?

Reply:
`axis=0` would provide a scale for each dimension. However, this would only make sense for an ellipse having semi-major/minor axis aligned with the axis of the coordinate system. In the case of an arbitrarily rotated ellipse, we would need to take the angle into account in the backward transformation. IMHO, this would add more complexity than giving benefits.

Reply:
Ah, that makes sense, then just leave it. Thanks for the insight.